WordPress/Classes/Services/KeyValueDatabase.swift

```diff
@@ -7,6 +7,7 @@ import Foundation
 /// list objects.
 protocol KeyValueDatabase {
     func object(forKey defaultName: String) -> Any?
+    func bool(forKey: String) -> Bool
```

Comment:
This should be in a protocol extension since it’s just a wrapper that will be the same for all cases

Reply:
Nice catch! Updated 👍